data/json/mapgen/cabin.json

```diff
@@ -127,6 +127,45 @@
       "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ] } ]
     }
   },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "cabin" ],
+    "weight": 333,
```

Comment:
I assume this weighting means that quite a few cabins will contain duplicates of this NPC ?